### DISCUSSION TITLE
请求更新，蔚蓝和以撒的结合重生 和地痞街区不要wine

### DISCUSSION THREAD

**Discussion by mmmgpx** [Gamuxorg/bbs#427] [投递和请求游戏]
需要蔚蓝 和，以撒的结合重生

**Comment by peckercode** [comment]
没有蔚蓝

**Comment by mmmgpx** [comment]
我春促的时候入了，如果要资源的话，可以找我

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by kero990** [comment]
蔚蓝在这里有https://www.linuxgame.cn/celeste
